loops-arrays-objects.js

```diff
@@ -130,6 +152,20 @@ const students = [
 //     { name: 'Karima', city: 'Utrecht', zipCode: '3531', neighborhood: 'Vogelenbuurt' }
 // ]
 // ==========================================
+for (let i = 0; i < students.length; i++) {
+    if (parseInt(students[i].zipCode) === 3512) {
+        students[i].neighborhood = "Vogelenbuurt";
+    }
+    else if (parseInt(students[i].zipCode) <= 3513) {
+        students[i].neighborhood = "Pijlsweerd";
+    } else if (parseInt(students[i].zipCode) > 3513) {
+        students[i].neighborhood = "Vogelenbuurt";
+
+    }
```

Comment:
Een switch gaat je wat code schelen (zeker als je alle postcodes zou willen verwerken)

Reply:
Hoe dan ? Want 3512 "Pijlsweerd" zit midden in de range van "vogelenbuurt". En wie weet zijn er meer kruizende wijken. Ik heb deze statement puur gemaakt op de aangeleverde data. Daarom zou ik in een Switch hetzelfde doen :

```
switch (true) {
    case parseInt(students[i].zipCode) === 3512:
        students[i].neighborhood = "Vogelenbuurt";
        break;
    case parseInt(students[i].zipCode) <= 3513:
        students[i].neighborhood = "Pijlsweerd";
        break;
    case parseInt(students[i].zipCode) > 3513:
        students[i].neighborhood = "Vogelenbuurt";
        break;
    default:
        // Default case if none of the conditions match
        break;
}
```
Of is er een andere, betere manier ? Ik ben benieuwd :)